src/engine/tools/engine-suite/src/engine_test/integration.py

```diff
@@ -0,0 +1,91 @@
+from engine_test.events_collector import EventsCollector
+from engine_test.formats.syslog import SyslogFormat
+from engine_test.formats.json import JsonFormat
+from engine_test.formats.eventchannel import EventChannelFormat
+from engine_test.formats.macos import MacosFormat
+from engine_test.formats.remote_syslog import RemoteSyslogFormat
+from engine_test.formats.audit import AuditFormat
+from engine_test.formats.command import CommandFormat
+from engine_test.formats.full_command import FullCommandFormat
+from engine_test.formats.multi_line import MultilineFormat
+from engine_test.event_format import Formats
+from engine_test.crud_integration import CrudIntegration
+
+from engine_test.api_connector import ApiConnector
+
+class Integration(CrudIntegration):
+    def __init__(self, args):
+        self.args = args
+
+        # Get the integration
+        integration_name = self.args['integration-name']
+        self.integration = self.get_integration(integration_name)
+        if not self.integration:
+            print("Integration not found!")
+            exit(1)
```

Comment:
What is the purpose of ending up with two different error codes? Do I check each case with the exit code?